examples/benchmarks/sim_speed_benchmark.py

```diff
@@ -53,6 +64,13 @@ def make_sim(
 
     return sim
 
+def warmup(sim, batch_size):
+    """Warmup the simulator."""
+
+    for episode_count in range(5):
+        sim.reset(list(range(batch_size)))
+        for _ in range(80):
+            sim.step()
```

Comment:
I don't think you can warmup for 80 steps? Most agents will have exited the scene by that point. Am I missing something?

Reply:
Teh episode length is 91. It doesnt matter how many step we make. We just want to do some steps before benchmarking to do warmup.